build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -192,11 +103,37 @@
 
       <CopyLocalBuildFile Include="@(CopyLocalBuildCrossTargetingFile)" />
       <CopyLocalBuildFile Include="@(CopyLocalSdkFile)" />
+
+      <!-- Use the contents of the output directory of the net46 build of the SDK tasks, except
+           for MSBuild and Roslyn DLLs and their dependencies-->
+      <Net46OutputAssembly Include="$(OutDir)net46\*.dll" />
```

Comment:
There shouldn't be any new or removed files as part of this change, just changes in which versions are used.  However, it would be good to have validation that we don't forget to update the .swr, so I've filed #689 for that.